### PR TITLE
create docker container. add fasterxml dependency to build.sbt

### PR DESCRIPTION
**Pull request by paulCodes**
basic docker container.

### CODE REVIEW

File: core/build.sbt [hunk 1]
```diff
@@ -20,6 +20,7 @@ libraryDependencies ++= Seq(
     "org.apache.geronimo.specs" % "geronimo-jms_1.1_spec" % "1.1.1",
     "org.reflections" % "reflections" % "0.9.9-RC1",
     "com.novocode" % "junit-interface" % "0.10" % "test",
+    "com.fasterxml.jackson.core" % "jackson-databind" % "2.5.1",
```
Comment (dcshock) [inline core/build.sbt]: Is there a reason this is being included with this PR?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Update system
+RUN apt-get update
+RUN apt-get upgrade -y
+
+# Install unzip
+RUN apt-get install unzip -y --force-yes --no-install-recommends
+
+# Add forklift server
+ADD ./server/target/universal/forklift-server-0.1.zip /tmp/forklift-server-0.1.zip
```
Comment (dcshock) [inline Dockerfile]: Can you add a doc file explaining how the 0.1.zip file is generated? Or better yet, the process to follow to generate the docker container?

File: core/build.sbt (hunk 1 above)
Comment (zdavep) [inline core/build.sbt]: The forklift server won't start in a docker container without it.  Errors out with a ClassNotFound exception.

Comment (dcshock) [inline core/build.sbt]: Got it. Thanks for the info.

File: Dockerfile (hunk 2 above)
Comment (paulCodes) [inline Dockerfile]: added DOCKERCONTAINER.md

**Pull request merged**